README.md

```diff
@@ -1 +1,116 @@
+    * The final point of this milestone is to make sure that you thought about all actions needed to play * Tic Tac Toe. It does not need to work yet - this will be done in the next milestone.
+  * <strong> Milestone 3: Game logic </strong>
+    * Suggested branch name game_logic.
+    * Add real logic for your game - get rid of all inline comments and mocked logic used in previous milestone.
+    * As main.rb file should be responsible for user interaction, this is the only(!) one place where puts/print and get.chomp can be used.
+    * Think how you can encapsulate diffrent parts of game logic in classes. Do not create one class for everything.
+    * Read about DRY rule and use it.
+    * Think about the diffrence between attr_readers/attr_writers and atrr_acessors. Use atrr_acessors only if you really need it.
+    * Remember about single responsibility rule. Always.
+    * Remember to cover edge cases: think what player could try to do to cheat the game and make sure that your program does not allow to perform invalid action.
+  * <strong>Milestone 4: Game instructions and final touches</strong>  
+    * Suggested branch name readme_game_instructions
+    * Add full description of game rules in README file - remember that some people might not be familiar with Tic Tac Toe game rules.
+    * Make sure that you informed your players on how to start your game - they might not be programmers.
+    * Improve anything you want to improve in order to mkae your game perfect.
+    
+    ![Project Screen Shot][product-screenshot]
+  
+### Built With
+This project was built using raw Ruby. 
+
+<!-- USAGE EXAMPLES -->
+## Use
+<bold>How should this be manually tested?</bold>
+Clone the repo then navigate to the project directory, on the terminal type bin/main.rb
+###### How to play ######
+* After choosing a name, Player 1 (X) will start.
+* Press 1 - 9 to place your move in the 3 x 3 grid that will show on screen.
+  * <br>![Numbered grid][numbers-grid]
+* After game ends, press y or n to continue or exit.
+* You can force a break in the loop by typing -5 as an input.
+
+<i>Tic-Tac-Toe instructions</i> 
+ * Tic-Tac-Toe is a paper-and-pencil game for two players, X and O, who take turns marking the spaces in a 3×3 grid.
+ * Player move is permanent and can't be moved again. If a position is taken you can not overwrite it.
+ * The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row is the winner.
+ * If no move is available, then it's a tie.
+ 
+<!-- ROADMAP -->
+## Roadmap
+
+See the [open issues ](https://github.com/bafiam/tic_tac_toe/issues)for a list of proposed features (and known issues).
+
+<!-- CONTACT -->
+## Contact
+
+Facundo Iglesias - [Github profile](https://github.com/Fig77)
+
+Stephen Gumba - [Github profile](https://github.com/bafiam)
+
+Project Link: - [Project repo](https://github.com/bafiam/tic_tac_toe)
+
```

Comment:
- [ ] Add a section for `RULES`. This [website](https://www.exploratorium.edu/brain_explorer/tictactoe.html) has well-written rules for the game.